scripts/index.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-use-before-define */
+const form = document.getElementById('form');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const list = document.getElementById('list');
+
+let books = [];
+
+function addingBookToHtml(Title, Author, index) {
+  const div = document.createElement('div');
+  div.className = 'list';
+  div.setAttribute('id', index);
+  const title = document.createElement('p');
+  title.innerHTML = Title;
+  title.className = 'bookTitle';
+  const author = document.createElement('p');
+  author.innerHTML = Author;
+  author.className = 'bookAuthor';
+  const button = document.createElement('button');
+  button.innerHTML = 'Remove';
+  button.className = 'RemoveButton';
+  button.setAttribute('id', 'removeButton');
+
+  button.addEventListener('click', ((e) => {
```

Comment:
- [ ] Kindly make sure to change the value of the books in `localStorage` because the data needs to be preserved as in the project requirements